App/Sagas/ThreadsSagas.ts

```diff
@@ -57,55 +56,19 @@ export async function getDefaultThread (): Promise<Thread> {
 
 export function * pendingInvitesTask () {
   // Process any pending external invites created while user wasn't logged in
-  const threads = yield select(ThreadsSelectors.threads)
-  if (threads.pendingInviteLink) {
-    yield call(DeepLink.route, threads.pendingInviteLink, NavigationService)
+  const pendingInviteLink: string | undefined = yield select(ThreadsSelectors.pendingInviteLink)
+  if (pendingInviteLink) {
+    yield call(DeepLink.route, pendingInviteLink, NavigationService)
     yield put(ThreadsActions.removeExternalInviteLink())
```

Comment:
Redid this in my existing PR, but didn't want to touch ThreadsSagas until this beast was in master so just worked around it for now

Reply:
The pending invite probably _does_ want to hit the AuthRedux, since it can just store the referral code right in the form data that way. and i con't see pending thread invites being used outside the auth flow, so... let's revisit after this merges